scripts/simnet.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+##################################################################
+# Starts the broker docker stack on simnet
+##################################################################
+
+set -e -u
+
+docker-compose up -d
```

Comment:
don't you still need prod?

Reply:
@treygriffith technically there are 2 versions of simnet. Hosted and Local. We have changes coming that will change the build process (using simnet instead of local dev) This is probably OK for now.